ApplyButterworthHighpass1D/CMakeLists.txt

```diff
@@ -40,6 +40,14 @@ SEMMacroBuildCLI(
   INCLUDE_DIRECTORIES ${MODULE_INCLUDE_DIRECTORIES}
   ADDITIONAL_SRCS ${MODULE_SRCS}
   )
+# Optional IPO. Do not use IPO if it's not supported by compiler.
+include(CheckIPOSupported)
+check_ipo_supported(RESULT result OUTPUT output)
+if(result)
+  set_property(TARGET ${MODULE_NAME} PROPERTY INTERPROCEDURAL_OPTIMIZATION TRUE)
+else()
+  message(WARNING "IPO is not supported: ${output}")
+endif()
 
```

Comment:
Should we add this to `SEMMacroBuildCLI` ?

Reply:
It requires CMake 3.13.0 to be supported with Visual Studio -- if that is OK for `SEMMacroBuildCLI`, then yes :-)

Reply:
Arf, CI is failing because of the CMake version requirement, too. I will make it optional based on CMake version, and contribute it to `SEMMacroBuildCLI`, too.